### DISCUSSION TITLE
Lesson 6: forking mainnet with Alchemy in Dev Env

### DISCUSSION THREAD

**Discussion by Decentralisedme** [smartcontractkit/full-blockchain-solidity-course-py#1457] [Q&A] [answered]
Hello, at the moment when you create a new app in Alchemy, the selection of the environment is not available anymore in the screen.
Please see screenshot below: new (top) screen vs the previous screen 
![image](https://user-images.githubusercontent.com/54667163/166643097-d0f93323-edfb-49a8-8f8a-16f0f096871e.png)
 
Is the cdm line still valid?
`brownie networks add development mainnet-fork-dev cmd=ganache-cli host=http://127.0.0.1 fork=https://eth-mainnet.alchemyapi.io/v2/c40U6PVX1zEFdcRfYVjXMC2X1RBpNKpx accounts:10 mnemonic=brownie port=8545`

To be noticed that in alchemy dashboard you only see App Name + Network, it does not show the environment anymore:
![image](https://user-images.githubusercontent.com/54667163/166645988-05b4df59-4805-4c87-a9fc-2ede407fbf49.png)

Does it mean I am in Mainnet "production"? 
Great course .... thank you very much!!!

**Comment by cromewar** [comment, marked as answer]
Hello @Decentralisedme 
Sorry for the delay I was not notified as usually by email with your question.
I just check  alchemy and you are right it does not have the option anymore, however it still working without problems.

Thanks for the tip ;)

**Reply by Decentralisedme** [reply]
Hello, no worries. 
Yes, thanks for confirming
